oracle_fdw.h

```diff
@@ -217,6 +217,7 @@ extern void oracleClientVersion(int *major, int *minor, int *update, int *patch,
 extern void oracleServerVersion(oracleSession *session, int *major, int *minor, int *update, int *patch, int *port_patch);
 extern void *oracleGetGeometryType(oracleSession *session);
 extern int oracleGetImportColumn(oracleSession *session, char *dblink, char *schema, char **tabname, char **colname, oraType *type, int *charlen, int *typeprec, int *typescale, int *nullable, int *key);
+extern int min_oracle_version(oracleSession *session, int major, int minor);
```

Comment:
I think that should be a static function in `oracle_fdw.c`.
It does not access the Oracle C API.

Reply:
No, you can not, there is a reference issue when calling fdwState->session->"any struct element" in oracle_fdw.c the only way I found is to declare the function in oracle_util.c, Honestly I have not taken the time to look forward at this problem.

Reply:
Ok, I'll look harder later when I have more time.

Reply:
I remember now - I had changed `oracleServerVersion` so that you can call that and don't need to create a new function.
See my pull request to your fork.